### PR TITLE
Cleanup the networking concepts page

### PR DESCRIPTION
**Pull request by tengqm**
This PR modifies the "what's next" section of the networking concept page. The pointer to the **archived** design proposals is only about the original design and it was archived two years ago. For people interested on "future plans", they should be redirected to the KEPs for SIG-Network.
This PR also fixes the line wrapping problems in two paragraphs (no content changes there).

### CODE REVIEW

**Review by Gauravpadam — COMMENTED** [review]
1. I like the manual line wrapping changes, This will make future changes easier to review
2. I think It's a good idea to add a reference for SIG-Network KEPs in the 'what's next' section

Overall LGTM

**Comment by windsonsea** [comment]
Thanks.
/lgtm

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -34,12 +34,21 @@ To learn about the Kubernetes networking model, see [here](/docs/concepts/servic
...
+for a non-exhaustive list of networking addons supported by Kubernetes.
 
 ## {{% heading "whatsnext" %}}
 
-The early design of the networking model and its rationale, and some future
-plans are described in more detail in the
+The early design of the networking model and its rationale are described in more detail in the
 [networking design document](https://git.k8s.io/design-proposals-archive/network/networking.md).
+For future plans and some on-going efforts that aim to improve Kubernetes networking, please
+refer to the SIG-Network
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/networking.md:52]: (nit) SIG Network.

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**